hedyorg/hedy
# 💻 Programs count

Fixes #5193 
Fixes #5192 
Fixes #1275

This fixes that only the programs which are modified are counted as (for now) programs saved.
[This](https://github.com/hedyorg/hedy/issues/5386) is the new issue for the design that shows this counter (where programs_saved will be changed):


**Also solved that only if a program is modified, the confetti button is shown:**

https://github.com/hedyorg/hedy/assets/48122190/dd0e6fe2-5484-4192-8f92-be01d1d5607e


How to test:
- Run a program with preview code, make sure it doesn't get added to my-programs
- Run a program with 'modified code' so not the preview code and make sure it does get added to my programs
- Remove a program from my programs and make sure the counter and the options in the forms get properly updated
